.builds/archlinux.yml

```diff
@@ -12,7 +13,7 @@ packages:
   - wayland
   - wayland-protocols
   - xcb-util-image
-  - xorg-server-xwayland
+  - xorg-xwayland
```

Comment:
I wonder if we need `xorg-xwayland` at all, actually. We don't link to it, it's purely a runtime dependency.

Reply:
I would have thought wlroots uses the xwayland.pc file for some of the Xwayland variables but it looks like it doesn't.

AFAICT the potentially useful vars provided are "have_initfd" which wlroots doesn't use but maybe could? (I'm not 100% sure what it's for) and "xwayland" which gives the path to the Xwayland binary, which wlroots could maybe make use of, but doesn't. The commit message here has some info about why we might want to use the pc file: https://gitlab.freedesktop.org/xorg/xserver/-/commit/5188603ff7d25186c9076fefeceba825f3a40d47

So I think we could drop it now, but might have to add it back in if we want to make use of those features in wlroots later. What do you think?

Reply:
>AFAICT the potentially useful vars provided are "have_initfd" which wlroots doesn't use but maybe could? (I'm not 100% sure what it's for) and "xwayland" which gives the path to the Xwayland binary, which wlroots could maybe make use of, but doesn't.

Yeah. We don't use it because we don't check for Xwayland at build-time. I don't know if we should.

>So I think we could drop it now, but might have to add it back in if we want to make use of those features in wlroots later. What do you think?

I guess it's fine to keep it, doesn't hurt.